<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PaxoTechWin7X64" name="ChromeScript" tests="9" failures="3" timestamp="28 Sep 2015 18:23:25 GMT" time="304.218" errors="0">
  <testcase name="SmSAlart" time="10.187" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
  <testcase name="boundaryInfo" time="17.06" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
  <testcase name="emtyInfologin" time="48.428" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 41.64 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &amp;apos;2.45.0&amp;apos;, revision: &amp;apos;5017cb8e7ca8e37638dc3091b2440b90a1d8686f&amp;apos;, time: &amp;apos;2015-02-27 09:10:26&amp;apos;
System info: host: &amp;apos;PaxoTechWin7X64&amp;apos;, ip: &amp;apos;10.10.30.41&amp;apos;, os.name: &amp;apos;windows&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.7.0_60&amp;apos;
Session ID: f919a615-427e-4aec-b318-007ebe6404f0
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=WINDOWS, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=37.0.2}]">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 41.64 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'PaxoTechWin7X64', ip: '10.10.30.41', os.name: 'windows', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_60'
Session ID: f919a615-427e-4aec-b318-007ebe6404f0
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=WINDOWS, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=37.0.2}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:268)
	at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:89)
	at com.paxotech.qa.lord.LoginBasePage.EmailAddress(LoginBasePage.java:16)
	at com.lordandTaylor.testsripts.LoginBaseFanctionality.emtyInfologin(LoginBaseFanctionality.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:391)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'PaxoTechWin7X64', ip: '10.10.30.41', os.name: 'windows', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_60'
Driver info: driver.version: unknown
	at <anonymous class>.fxdriver.cache.getElementAt(resource://fxdriver/modules/web-element-cache.js:8953)
	at <anonymous class>.Utils.getElementAt(file:///C:/Users/Paxo/AppData/Local/Temp/anonymous1839356003951834713webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:8546)
	at <anonymous class>.fxdriver.preconditions.visible(file:///C:/Users/Paxo/AppData/Local/Temp/anonymous1839356003951834713webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:9585)
	at <anonymous class>.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/Paxo/AppData/Local/Temp/anonymous1839356003951834713webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12257)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/Paxo/AppData/Local/Temp/anonymous1839356003951834713webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12274)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/Paxo/AppData/Local/Temp/anonymous1839356003951834713webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:603)
]]>
    </failure>
  </testcase> <!-- emtyInfologin -->
  <testcase name="invalidEmailAddforReg" time="19.542" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
  <testcase name="invalidRegistration" time="20.728" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
  <testcase name="invlaidUerLogin" time="10.052" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]&amp;gt; but was:&amp;lt;[[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]]&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]> but was:<[[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]]]]><![CDATA[>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.paxotech.qa.lord.LoginBasePage.ivalidEamil(LoginBasePage.java:42)
	at com.lordandTaylor.testsripts.LoginBaseFanctionality.invlaidUerLogin(LoginBaseFanctionality.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:391)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- invlaidUerLogin -->
  <testcase name="newMemberRegistration" time="16.096" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
  <testcase name="verifyhomepagefaunctionality" time="5.783" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality">
    <failure type="java.lang.AssertionError" message="expected [] but found [Lord and Taylor: Designer Clothing, Shoes, Handbags, Accessories &amp;amp; More]">
      <![CDATA[java.lang.AssertionError: expected [] but found [Lord and Taylor: Designer Clothing, Shoes, Handbags, Accessories & More]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at com.paxotech.qa.lord.HomeBasePage.veriftyTitle(HomeBasePage.java:145)
	at com.lordandTaylor.testsripts.LoginBaseFanctionality.verifyhomepagefaunctionality(LoginBaseFanctionality.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:391)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- verifyhomepagefaunctionality -->
  <testcase name="vlaidUerLogin" time="14.826" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
</testsuite> <!-- ChromeScript -->
